release-notes/opensearch-skills.release-notes-2.13.0.0.md

```diff
@@ -0,0 +1,9 @@
+# 2024-03-20 Version 2.13.0.0
+
+Compatible with OpenSearch 2.13.0
+
```

Comment:
We should also mention the [dependencies](https://github.com/opensearch-project/skills/commits/2.13/) bump as under `Upgrades`

Reply:
They're not critical and not features for customers.

Reply:
We do that for [OpenSearch](https://github.com/opensearch-project/OpenSearch/blob/main/release-notes/opensearch.release-notes-2.12.0.md#dependencies) but upto you.

Reply:
>We should also mention the [dependencies](https://github.com/opensearch-project/skills/commits/2.13/) bump as under Upgrades

+1 on this. We should add all the PRs in the release notes. May be in a different section like maintenance/upgrades

Reply:
Let's merge this first and I can create another PR to add these.